src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -298,7 +298,7 @@ export const NewProposal = ({ navigation, data, query }) => {
               control={control}
               render={(field) =>
                 item.name === 'image' ? (
-                  <ImageSelector {...{ control, field, item }} />
+                  <ImageSelector {...{ control, field, item, imageId: data?.imageId }} />
```

Comment:
aaahhh this is the `imageId` i was questioning in the other PR, right? https://github.com/ikuseiGmbH/smart-village-app-app/pull/539#discussion_r884953915